### PR TITLE
added comments to vectors.dart

### PR DESCRIPTION
**Pull request by llllOllOOll**
I added comments in English and I would like to know how I went in this first time.

This is my second, Pull request and I want to describe how to do that:

git pull // Get last updates from origin/master
make changes and don't commit yet
git checkout -b update_somethings origin/master
git add .
git commit -m 'added new changes'
git push origin update_somethings

### CODE REVIEW

File: example/vectors.dart [hunk 1]
```diff
@@ -1,11 +1,46 @@
...
+
+// How to use vector to move objects on games and animation.
+// The main function of vectors in game is control velocity and direction.
+// We can use it to determine the distance between elements. (Vector length)
+//
+// Inversion of vectors
+//
+// We can invert a vector in the simple way, where is head becames tail and tail becames head
+// To do that  in programming we just add -1  to the vector.
+
```
Comment (ToniCalfim) [inline example/vectors.dart]: Only testing the multi-line comment.

File: example/vectors.dart [hunk 2]
```diff
@@ -1,11 +1,46 @@
...
+ *
+ * Math for game developers
+ * 
+ * 
+ * 
+ * 
+ *
+ * 
+ *
+ *
```
Comment (ToniCalfim) [inline example/vectors.dart]: Pra que esse tanto de linha em branco?

File: example/vectors.dart [hunk 3]
```diff
@@ -1,11 +1,46 @@
...
+// We can use it to determine the distance between elements. (Vector length)
+//
+// Inversion of vectors
+//
+// We can invert a vector in the simple way, where is head becames tail and tail becames head
+// To do that  in programming we just add -1  to the vector.
+
 import 'dart:html';
 
+// Create a Canvas and added it to the DOM  and get 2D context
```
Comment (ToniCalfim) [inline example/vectors.dart]: added = add

File: example/vectors.dart (hunk 1 above)
Comment (llllOllOOll) [reply example/vectors.dart]: This doesn't make any senses!!

File: example/vectors.dart [hunk 4]
```diff
@@ -52,15 +109,26 @@ class Point {
...
-    potision.y = y + vector.y;
+    position.x = x + vector.x;
+    position.y = y + vector.y;
 
-    return potision;
+    return position;
   }
 }
 
+// Subtracting two points / vectors
```
Comment (ToniCalfim) [inline example/vectors.dart]: "Subtracting" but / signal used. This is very unclear!

File: example/vectors.dart (hunk 2 above)
Comment (llllOllOOll) [reply example/vectors.dart]: I will fix it. I would like to add more information here.

File: example/vectors.dart [hunk 5]
```diff
@@ -52,15 +109,26 @@ class Point {
...
+// return a new vector with a result of the subtract.
```
Comment (ToniCalfim) [inline example/vectors.dart]: "...with the result of the subtraction."

File: example/vectors.dart (hunk 4 above)
Comment (llllOllOOll) [reply example/vectors.dart]: What I meant we can subtracting  two points or two vectors. What do you think about that: Subtracting two both vectors and points.

File: example/vectors.dart (hunk 5 above)
Comment (llllOllOOll) [reply example/vectors.dart]: It will return the result of subtraction.

File: example/vectors.dart (hunk 4 above)
Comment (ToniCalfim) [reply example/vectors.dart]: "Subtracts two points and returns the vector."

**Pull request merged**